frontend/packages/core/src/Theme/types.tsx

```diff
@@ -41,3 +41,12 @@ export interface ClutchColors {
 }
 
 export type ComponentState = "hover" | "focused" | "pressed" | "selected" | "disabled";
+
+export enum ThemeSpacing {
+  None = 0,
+  ExtraSmall = 0.5,
+  Small = 1,
+  Base,
+  Medium,
+  Large,
```

Comment:
What do these represent?

Reply:
I was going for generic names for the spacing variants as using something like `PageGutter` could be self-descriptive, but the value could be repeated somewhere else.

Reply:
Do these map to something already? Curious why some are given a value while the others aren't. I haven't looked at the spacing variants for mui so maybe it's explained there.

Reply:
Not really a map, these values, which are in order 0, 0.5, 1, 2, 3 and 4, are used as a multiplier of the scaling factor to produce a value, so this enum is just a giving them a name.

It's expected to apply the enum members as following examples:

```diff
<Grid
- spacing={2}
+ spacing={ThemeSpacing.Base}
>
  <Component />
 </Grid>
 ```
 
 ```diff
const StyledComponent = styled(Component)(({ theme }: { theme: Theme }) => ({
- margin: "24px",
+ margin: theme.spacing(ThemeSpacing.Medium),
}));
 ```
 
 There's an example in [MUI's docs](https://v5.mui.com/material-ui/customization/spacing/#custom-spacing) talking about the custom spacing.